robot/web/web_c.go

```diff
@@ -109,15 +110,12 @@ func (svc *webService) initStreamServerForModule(ctx context.Context) error {
 	}
 
 	// Register the stream server + APIs with the gRPC server for modules.
-	if err := svc.modServer.RegisterServiceServer(
+	return srv.RegisterServiceServer(
```

Comment:
same question as above, are there concurrency / locking issues to worry about now